docs/getting_started/dashboard.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Console Dashboard
+sidebar_position: 3
+---
+
+The [Console](https://app.calimero.network/dashboard) dashboard is your gateway to managing your Calimero Private Shard. Here you can access all the information about your nodes, indexers and services. The dashboard also provides users with access to important information such as the RPC and Indexer GraphQL URL endpoints, as well as basic statistics about the network, including current block height, number of transactions, and number of accounts.
+
+If your infrastructure needs grow, you can use the Console to scale up the number of validators, RPCs, and indexers running on your network. This ensures that your network can handle increased traffic as your business grows.
+
+- **Environment**: This refers to either Testnet or Mainnet blockchain environment.
+- **Account**: This refers to the total number of account you have in each shard
+- **Validators**: This refers to the number of active validator node currently participating in the consensus process of validating transactions and creating new blocks in the blockchain network.
+- **Block height**:  This refers to the number of blocks that have been added to the blockchain within that specific private shard.
+- **Transactions**: This refers to all transactions(_FUNCTION_CALL, CREATE_ACCOUNT, ADD_KEY, TRANSFER_ etc) you have made in the shard.
```

Comment:
```suggestion
- **Transactions**: This refers to all transactions(_FUNCTION_CALL, CREATE_ACCOUNT, ADD_KEY, TRANSFER_ etc) executed in the shard.
```
The number is not personalized, and there the console account is not linked to the blockchain account